config/config.dhall

```diff
@@ -5,20 +5,20 @@ Comments other than here will be stripped out because of how the Dhall auto-form
 This config should not be checked into git with passwords and other sensitive values saved.
 
 -}
-let LogLevel = ./config/LogLevel.dhall
+let LogLevel = ./lib/LogLevel.dhall
```

Comment:
I moved all the config stuff into a directory so it was easy to mount using the compose.yaml starting from this directory